source/ide/ide_methods.bas

```diff
@@ -18908,29 +18908,44 @@ FUNCTION ideupdatehelpbox
         SELECT CASE UpdateStep
             CASE 1
                 'Create a list of all files to be recached
+                st# = TIMER(0.001)
                 IF Help_Recaching < 2 THEN
                     f$ = CHR$(0) + idezfilelist$("internal/help", 2, "*.txt") + CHR$(0)
                     IF LEN(f$) = 2 THEN f$ = CHR$(0)
                 ELSE
                     f$ = CHR$(0) 'no dir scan for 'qb64pe -u' (build time update)
                 END IF
-
-                'Prepend core pages to list
-                f$ = CHR$(0) + "Keyword_Reference_-_By_usage.txt" + f$
-                f$ = CHR$(0) + "QB64_Help_Menu.txt" + f$
-                f$ = CHR$(0) + "QB64_FAQ.txt" + f$
+                et# = TIMER(0.001) - st#: IF et# < 0 THEN et# = et# + 86400
+                IF 1.25 - et# > 0 THEN _DELAY 1.25 - et#

```

Comment:
What purpose is the delay for? Does that avoid getting errors from `qb64phoenix.com` or something to that effect?